### PR TITLE
make docs command

### PR DESCRIPTION
**Pull request by ferminho**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,82 @@
...
+Nextail Terraform module for creating a Databricks workspace in AWS within an existing VPC
+
+<!-- BEGIN_TF_DOCS -->
+## Requirements
+
+| Name | Version |
+|------|---------|
+| <a name="requirement_terraform"></a> [terraform](#requirement\_terraform) | >= 1.1.2 |
+| <a name="requirement_aws"></a> [aws](#requirement\_aws) | >= 3.37.0 |
+| <a name="requirement_databricks"></a> [databricks](#requirement\_databricks) | >= 0.5.7 |
```
Comment (XaviPolaNextail) [inline README.md]: Isn't this the version we want to move forward?

File: README.md [hunk 2]
```diff
@@ -1,4 +1,82 @@
...
+
+## Providers
+
+| Name | Version |
+|------|---------|
+| <a name="provider_aws"></a> [aws](#provider\_aws) | >= 3.37.0 |
+| <a name="provider_databricks"></a> [databricks](#provider\_databricks) | >= 0.5.7 |
```
Comment (XaviPolaNextail) [inline README.md]: Version shouldn't be >=1.0.0?

File: README.md (hunk 1 above)
Comment (ferminho) [reply README.md]: Yes, but the change is not on main (yet). The intention of the PR, if approved, is to use this new command in the other branch to update the other PR docs

Comment (ferminho) [reply README.md]: updated!

File: README.md (hunk 2 above)
Comment (ferminho) [reply README.md]: it is now :+1:

**Comment by ferminho** [comment]
Merged with latest and fixed remaining issue

**Review by XaviPolaNextail — APPROVED** [review]
👍

**Comment by ferminho** [comment]
I didn't sign my commits. Re-creating PR

**Pull request closed without merging**